app/src/main/java/com/example/composetemplate/presentation/common/LoginTextField.kt

```diff
@@ -49,24 +47,24 @@ fun LoginTextField(
                     else {
                         if (!isValid) Color.Red else Color.Transparent
                     }
-                ), CircleShape
+                ), CustomTheme.shapes.textField
```

Comment:
is textField the right name for circle shape? what if there are textfield that dont need circle shape?

Reply:
That's the preferred shape for the text field in the app. It represents the text field shape, not a circular shape. If the text field becomes rectangular in the future, I will update the shape accordingly.

Reply:
I will change to circleTextFiled